### ISSUE TITLE
Address `NOTE` in CRAN daily checks

### ISSUE THREAD

**Issue by IndrajeetPatil** [r-lib/lintr#1645]
Ref: https://www.stats.ox.ac.uk/pub/bdr/donttest/lintr.out

```r
* checking for new files in some other directories ... NOTE
Found the following files/directories:
  ‘~/.cache/R/lintr/6f33e1e449dedb5369c6de9553905934e4b097d7’
```

I think we should submit 3.0.2 soon (#1476). We have added a significant amount of functionality since the last release, in addition to numerous bug fixes.

cc @jimhester

**Comment by IndrajeetPatil** [comment]
Also, there are failures on Windows: https://cran.r-project.org/web/checks/check_results_lintr.html

But not sure if these are false positives. But they have been there since the release, so maybe there is something to it.

**Comment by jimhester** [comment]
I sent a patch release with just the changes needed to fix this yesterday.

**Comment by MichaelChirico** [comment]
Hi @jimhester I see the patch still hasn't landed on CRAN. If it's possible, we may just want to send a full 3.0.2 release. I think the current main is basically ready.

**Comment by jimhester** [comment]
CRAN had imposed a deadline and I didn't want to complicate the release more than it needed to be. We can do another release in the future once this one is on CRAN.

**Comment by MichaelChirico** [comment]
SG, thanks for handling this!

**Comment by IndrajeetPatil** [comment]
lintr 3.0.2 is on [CRAN](https://cran.r-project.org/web/packages/lintr/index.html).

Keeping this issue open to remind ourselves that we need to create a new release on GitHub.
